librosa/core/constantq.py

```diff
@@ -1478,17 +1491,24 @@ def griffinlim_cqt(
     )
 
 
-def __bpo_to_alpha(bins_per_octave: int) -> float:
-    """Compute the alpha coefficient for a given number of bins per octave
+def __et_relative_bw(bins_per_octave: int) -> np.ndarray:
```

Comment:
not sure why `_relative_bandwidth` and `__et_relative_bw` are not held to the same privacy status here (one vs. two leading underscores)

Reply:
It's only because relative_bandwidth may be called from a separate submodule but et_bandwidth can't.